src/Repository/UnleashRepository.php

```diff
@@ -12,4 +12,6 @@ public function findFeature(string $featureName): ?Feature;
      * @return iterable<Feature>
      */
     public function getFeatures(): iterable;
+
+    public function refreshCache(): void;
```

Comment:
This would mean a breaking change, it needs to be added as a phpdoc block above the interface like this:

```php
/**
 * @method void refreshCache()
 */
```

Afterwards, when a new major version is released, it will be moved to a real method.